schema/bom-1.6.schema.json

```diff
@@ -2929,11 +2929,11 @@
       "minLength": 1,
       "maxLength": 1024,
```

Comment:
Can we remove maxLength or increase it further to 4096? Some git-based repos use commit hashes.

Reply:
Done. I've changed it to 4096